Misc/NEWS.d/next/IDLE/2019-09-01-10-22-55.bpo-35771.tdbmbP.rst

```diff
@@ -0,0 +1,2 @@
+Increase the ``hover_delay`` in IDLE's test_tooltip, and run both tests
+waiting for a delay to expire simultaneously, to reduce total run time.
```

Comment:
I changed this to what users are more likely to want to know.  "To avoid occasional spurious test_idle failures on slower machines,  increase the ``hover_delay`` in test_tooltip."